scipy/optimize/zeros.py

```diff
@@ -233,6 +233,21 @@ def bisect(f, a, b, args=(),
         Object containing information about the convergence.  In particular,
         ``r.converged`` is True if the routine converged.
 
+    Examples
+    --------
+    The bisection algorithm fails if `f(a)` and `f(b)` have the same sign.
```

Comment:
If you're formatting code snippets, those should have double rather than single backticks.